src/locales/en_US/en_US.json

```diff
@@ -524,5 +524,12 @@
         "next_button": "Choose document",
         "subtitle": "It should take a few minutes",
         "title": "Verify your identity"
+    },
+    "msvc_qr_code": {
```

Comment:
Also, make sure this is placed alphabetically within the file, as that appears to be the convention here.